Data/LookupTable.cs

```diff
@@ -8,19 +8,28 @@ namespace BrackeysBot
     /// <summary>
     /// Provides a lookup table that will be serialized into a JSON file.
     /// </summary>
-    public class LookupTable<TKey, TValue> : ILookupTable<TKey, TValue>
+    public abstract class LookupTable<TKey, TValue> : ILookupTable<TKey, TValue>
     {
-        [JsonProperty("Lookup")]
-        protected Dictionary<TKey, TValue> _lookup;
+        /// <summary>
+        /// Returns the path of the lookup file.
+        /// </summary>
+        public string FilePath => FileName + FILETYPE;
```

Comment:
The `FILETYPE` member shouldn't contain the period. Either make it like this:
```cs
public string FilePath => $"{FileName}.{FILETYPE}";
```
or rename it to `FILE_EXTENSION`.